presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -485,7 +486,7 @@ private void startInBackground(QueryExecution query)
                 group = group.parent.get();
             }
             updateEligiblility();
-            executor.execute(() -> query.start(Optional.of(id.toString())));
+            executor.execute(() -> query.start());
```

Comment:
Sorry, I just meant that given you basically reverted somebody else's diff, it would be good to have an explanation for the refactor in the commit message.